src/Sentry/PlatformAbstractions/RuntimeInfo.cs

```diff
@@ -82,6 +82,8 @@ internal static Runtime WithAdditionalProperties(this Runtime runtime)
                 version = frameworkInstallation?.Version?.ToString();
             }
         }
+#elif TRIMMABLE
+    private static string? GetNetCoreVersion(Runtime runtime) => null; // There is no runtime for AOT apps
```

Comment:
The comment seems misleading; there is runtime info, see

https://sentry-sdks.sentry.io/issues/4537780610/events/fae662afea604b7593d35d3ef7ae09af/?project=5428537&query=is%3Aunresolved+os.name%3AWindows&referrer=previous-event&statsPeriod=14d&stream_index=1#context-runtime

Reply:
But that's just the runtime that things were compiled with right? And if we bake that into our library, all it tells us is the runtime that the Sentry NuGet packages were compiled with (the user application that is making use of our SentrySDK might be built using a more recent runtime). the "actual" runtime is just native assembly for AOT apps right?

Reply:
I don't believe so. AFAICT the sentry.dll is built as a standard library and is only later packaged into the AOT published app.